src/main/webapp/js/changeStatus.js

```diff
@@ -0,0 +1,15 @@
+function changeStatus(id, element) {
+    const active = element.closest('ul').querySelector('#active-' +id);
+    const url = '/atualizarStatus?id=' + id
+
+    let xhr = new XMLHttpRequest();
```

Comment:
Detalhe somente, mas aqui poderia ser `const`:

```suggestion
    const xhr = new XMLHttpRequest();
```